analyzer/src/main/kotlin/managers/PhpComposer.kt

```diff
@@ -118,19 +118,26 @@ class PhpComposer : PackageManager() {
                 Files.move(vendorDir.toPath(), tempVendorDir.toPath(), StandardCopyOption.ATOMIC_MOVE)
             }
 
-            installDependencies(workingDir)
+            val manifest = jsonMapper.readTree(definitionFile)
+            val hasDependencies = manifest.fieldNames().asSequence().any { it.startsWith("require") }
```

Comment:
Have you tested a `composer.json` with `"requires": []` inside? I assume that would not produce a lock file either.

Reply:
Thanks for pointing out that case, I've added a new commit to support it.